port/common/omrport.tdf

```diff
@@ -1083,3 +1083,7 @@ TraceEvent=Trc_PRT_retrieveLinuxMemoryStats_CgroupMemoryStatsFailed Group=sysinf
 
 TraceEntry=Trc_PRT_signal_omrsig_is_signal_ignored_entered Group=signal Overhead=1 Level=3 NoEnv Template="omrsig_is_signal_ignored: Entered, portLibrarySignalFlag=0x%X"
 TraceExit=Trc_PRT_signal_omrsig_is_signal_ignored_exiting Group=signal Overhead=1 Level=3 NoEnv Template="omrsig_is_signal_ignored: Exiting, rc=%d, isSignalIgnored=%d"
+
+TraceException=Trc_PRT_signal_unblockSignals_sigemptyset_failed Group=signal Overhead=1 Level=3 NoEnv Template="omrsignal unblockSignals: sigemptyset failed with rc=%d and errno=%d"
```

Comment:
These are all exception paths so they can be Level=1

Reply:
updated to Level=1.